cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-deployment.adoc

```diff
@@ -0,0 +1,161 @@
++
+[source,terminal]
+----
+$ oc login --token=<your_token> --server=https://api.osd4-demo.abc1.p1.openshiftapps.com:6443
+Logged into "https://api.myrosacluster.abcd.p1.openshiftapps.com:6443" as "rosa-user" using the token provided.
+
+You don't have any projects. You can try to create a new project, by running
+
+oc new-project <project name>
+----
+
+=== Creating a new project
+
+.Procedure
+
+. Create a new project named `ostoy` in your cluster:
+
+.. To create a new project with the default project name, run following command from the CLI:
++
+[source,terminal]
+----
+oc new-project ostoy
+----
++
+.Example output
++
+[source,terminal]
+----
+Now using project "ostoy" on server "https://api.myrosacluster.abcd.p1.openshiftapps.com:6443".
+----
+
+.. If you require a unique project name, run following command:
++
+[source,terminal]
+----
+ oc new-project ostoy-$(uuidgen | cut -d - -f 2 | tr '[:upper:]' '[:lower:]')
+----
+
+. Alternatively, you can create a new project with the web console:
+
+.. From the web console, click *Home* -> *Projects*. 
+
+.. On the *Projects* page, click create *Create Project*.
++
+image::4-createnewproj.png[The project creation screen]
+
+=== Deploying the backend microservice
+
+The microservice serves internal web requests and returns a JSON object containing the current hostname and a randomly generated color string.
+
+. From your terminal, deploy the microservice with the following command:
++
+[source,terminal]
+----
+oc apply -f https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/yaml/ostoy-microservice-deployment.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/yaml/ostoy-microservice-deployment.yaml
+deployment.apps/ostoy-microservice created
+service/ostoy-microservice-svc created
+----
+
+=== Deploying the front-end service
+
+The front-end deployment uses the Node.js front-end for the application along with a few other Kubernetes objects.
+
+If you open the `ostoy-frontend-deployment.yaml`, you will see we are defining:
+
+- Persistent Volume Claim
+- Deployment Object
+- Service
+- Route
+- Configmaps
+- Secrets
+
+. Deploy the application front-end and create all of the objects by entering the following command:
++
+[source,terminal]
+----
+oc apply -f https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/yaml/ostoy-frontend-deployment.yaml
+----
++
+.Example output
++
+[source,terminal]
+----
+persistentvolumeclaim/ostoy-pvc created
+deployment.apps/ostoy-frontend created
+service/ostoy-frontend-svc created
+route.route.openshift.io/ostoy-route created
+configmap/ostoy-configmap-env created
+secret/ostoy-secret-env created
+configmap/ostoy-configmap-files created
+secret/ostoy-secret created
+----
++
+You should see all objects created successfully
```

Comment:
```suggestion
You should see all objects created successfully.
```
Alternatively, you could just delete this sentence. With the "Example output" above, it isn't really necessary.